src/zope/interface/_zope_interface_coptimizations.c

```diff
@@ -573,7 +611,8 @@ __adapt__(PyObject *self, PyObject *obj)
     {
       PyObject *implied;
 
-      implied = inst_attr(decl, str_implied);
+      //implied = ((Spec*)decl)->_implied;
+      implied = PyObject_GetAttrString(decl, "_implied");
```

Comment:
?

Reply:
Whoops. Sorry. I was checking to see if there were any tests in place that would detect the ref leak. Didn't mean to leave that in.